recipe/build.sh

```diff
@@ -13,6 +13,9 @@ set -x -e
 INCLUDE_PATH="${PREFIX}/include"
 LIBRARY_PATH="${PREFIX}/lib"
 
+# Always build PIC code for easier static linking
+CXXFLAGS="${CXXFLAGS} -O3 -fPIC"
```

Comment:
Don't get me wrong. If it works reliably, we definitely should go for that speed up. Just want to be sure about the reliability part.